src/nvector/serial/fmod/fnvector_serial_mod.f90

```diff
@@ -750,20 +750,20 @@ function FN_VGetArrayPointer_Serial(v) &
 use, intrinsic :: ISO_C_BINDING
 real(C_DOUBLE), dimension(:), pointer :: swig_result
 type(N_Vector), target, intent(inout) :: v
-type(C_PTR) :: fresult 
-type(C_PTR) :: farg1 
+type(C_PTR) :: fresult
+type(C_PTR) :: farg1
 
 farg1 = c_loc(v)
 fresult = swigc_FN_VGetArrayPointer_Serial(farg1)
-call c_f_pointer(fresult, swig_result, [1])
+call c_f_pointer(fresult, swig_result, [FN_VGetLength_Serial(v)])
```

Comment:
`N_VGetLocalLength_MPIPlusX` is missing from the `.h` file

Reply:
Comment on the wrong file? Or do you mean `N_VGetLocalLength_Serial`? I thought we decided we dont need it?

Reply:
Nope, just misread the function name. This is good.